lib/test.js

```diff
@@ -106,8 +107,11 @@ Test.prototype.test = function (name, opts, cb) {
     });
 };
 
-Test.prototype.comment = function (msg) {
+Test.prototype.comment = function () {
     var that = this;
+    var keys = Object.keys(arguments)
+    var msg = keys.length > 1 ? format.apply(null, arguments) : arguments[keys[0]];
```

Comment:
- just `arguments.length` is sufficient, no need to `Object.keys` (also that makes it require ES5, which this lib does not)
 - It seems like this might be easier to just `format.apply(null, arguments)` in every case?